sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -16086,6 +16086,7 @@ public void testScanAndSortOnJoin()
         .run();
   }
 
+  @NotYetSupported(Modes.UNSUPPORTED_DATASOURCE)
```

Comment:
hmm.. I wonder how did this started to happen?
its not a blocker just interested

Reply:
Not very sure though, but I expected this was because we used to send empty to window operator query and that has been changed now. Let me check the actual difference and comment back.